core/constants.go

```diff
@@ -445,3 +445,7 @@ const CommitMaxTime = time.Second
 
 // DefaultUnstakedEpoch represents the default epoch that is set for a validator that has not unstaked yet
 const DefaultUnstakedEpoch = math.MaxUint32
+
+// IntervalBlackListPeerInvalidMessage time to keep a peer in the black list if it sends a message that does not follow
```

Comment:
`represents the time...`

Reply:
also maybe rename to `InvalidMessageBlacklistDuration` or something similar. It's not an interval.

Reply:
done, renamed to InvalidMessageBlacklistDuration